p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -2556,7 +2589,8 @@ expression
     ;
 
 expressionList
-    : expression
+    : /* empty */
```

Comment:
@mbudiu-vmw Just checking, we weren't relying on `expressionList` being non-empty elsewhere?